### ISSUE TITLE
Audio is not being captured while publishing Sumerian scene with Amplify

### ISSUE THREAD

**Issue by tolivestone** [aws-amplify/amplify-js#5946]
I have implemented hands free solutions(followed steps giving the below link) in Sumerian and privately published via Amplify but it's not capturing audio. I see below error in console 

AudioContext.js:36 The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu 

https://docs.sumerian.amazonaws.com/articles/hands-free-voice-transcription/

**Comment by ashika01** [comment]
@jordanranz any thoughts?

**Comment by jordanranz** [comment]
Hey @tolivestone, are you using `XR.loadScene` for this scene or a UI component?

**Comment by tolivestone** [comment]
> Hey @tolivestone, are you using `XR.loadScene` for this scene or a UI component?

@jordanranz  Yes

`async loadAndStartScene() {
    await XR.loadScene(this.props.scene, "sumerian-scene-dom-id");
    const controller = XR.getSceneController(this.props.scene);
    this.props.onLoaded(controller);
    XR.start(this.props.scene);
  }`

**Comment by SavagePencil** [comment]
This is the autoplay-blocking enforced by some browsers.  It will require the customer to do a physical input before audio will function.  You may need a start button or other similar input to get past Chrome's blocking of audio.

**Comment by jordanranz** [comment]
Yep, as @SavagePencil mentioned (and highlighted in that Google link) user interaction is required to enable audio. There is an API on the XR category that is implement to do this:
https://github.com/aws-amplify/amplify-js/blob/master/packages/xr/src/Providers/SumerianProvider.ts#L311
```
XR.enableAudio
```

There is also an event that is fired on the scene that will tell you when audio is disabled, here is an example:
https://github.com/aws-amplify/amplify-js/blob/master/packages/aws-amplify-react/src/XR/SumerianScene.tsx#L136-L138
